contrib/cirrus/ooe.sh

```diff
@@ -22,12 +22,9 @@ OUTPUT_TMPFILE="$(mktemp -p '' $(basename $0)_output_XXXX)"
 output_on_error() {
     RET=$?
     set +e
-    if [[ "$RET" -ne "0" ]]
-    then
-        echo "---------------------------"
-        cat "$OUTPUT_TMPFILE"
-        echo "[$(date --iso-8601=second)] <exit $RET> $COMMAND"
-    fi
+    echo "---------------------------"
+    cat "$OUTPUT_TMPFILE"
+    echo "[$(date --iso-8601=second)] <exit $RET> $COMMAND"
     rm -f "$OUTPUT_TMPFILE"
```

Comment:
FYI- ``ooe.sh`` is only used during image builds.  Since they happen in parallel, the output gets overwhelming unless some things are masked (e.g. package installs).